### DISCUSSION TITLE
После активации данного метода обхода не запускается игра Call of Duty

### DISCUSSION THREAD

**Discussion by DarkTrailDog** [Flowseal/zapret-discord-youtube#590] [General]
Античит просто не запускает игру, если деактивировать [zapret-discord-youtube](https://github.com/Flowseal/zapret-discord-youtube), то игра запускается как ни в чем не бывало. Можно как то решить данный конфликт между античитом и обходом блокировки?

**Comment by Flowseal** [comment]
Легально - нет.

**Reply by DarkTrailDog** [reply]
> Легально - нет.

Вопрос не в этом. Я указал только на то, что при работе обхода, не запускается игра, если офнуть обход запрета, то все работает. Я так к сведению написал, так как многие игроки, кто пользуется обходом, не могут играть и ищут проблему из-за которой игра просто не стартует. А что легально, что нет, мне побарабану)

Да и вопрос в целом был, можно ли решить данный вопрос с запуском игры и рабочим обходом, а не про легально это или нет)
